taskmanager/features/task.feature

```diff
@@ -9,3 +9,12 @@
       When I create a task named "Add switch language button"
       Then there should be "1" tasks with "TODO" status
 
+    Scenario: Assigned unassigned task
+      Given there is unassigned task
+      When I assigned task to "Carl"
+      Then task should be assigned to "Carl"
```

Comment:
`Then task called "Add datepicker to birthdate field" should be assigned to "Carl"`
`And task should disappeared from the list of unassigned tasks`